libproto/README.md

```diff
@@ -0,0 +1,29 @@
+## Overview
+
+Provide the type of interaction between the cita-node 
+interaction type and the cita component, use protobuf 
+to generate the corresponding **.rs file and the 
```

Comment:
`**` is a special character.

Reply:
generate the corresponding `**.rs` which include types and some set/get methods.